conf/ocs_version/ocs-4.4.yaml

```diff
@@ -0,0 +1,10 @@
+---
+DEPLOYMENT:
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-olm-operator:latest-4.4"
+  default_latest_tag: 'latest-4.4'
+  ocs_csv_channel: "stable-4.4"
+ENV_DATA:
+  logging_version: '4.4'
```

Comment:
I think this is related to the OCP version .
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/templates/openshift-infra/logging-deployment/elasticsearch_operator/eo-sub.yaml

But would be worth to check with someone who did this logging.

And if it's related to the OCP version I guess it should be deleted from this OCS related config and should be moved to the OCP related config file.

So I think that maybe if you will just delete this and let the default value use cause I guess you will fail with deployment cause of missing 4.4 channel. But it's just my guess.

Reply:
I think you are right, maybe keep it as is which is '4.3' so that the tests pass.

Reply:
Please remove this value from here. This is also going to be deleted here:  https://github.com/red-hat-storage/ocs-ci/pull/1720/files